File: js/src/formulas/infiniteSeries4.ts
export default function infiniteSeries4 (x: number) {
  if (x > 1 || x < -1 || x === 1 || x === -1) {
    console.log('Check |x| < 1')
  } else {
    return (x / ((1 - x) ** 2))
  }
}

File: cs/src/formulas/infiniteSeries4.cs
public static class InfiniteSeries4
{
    public static double Calculate(double x)
    {
        if (x > 1 || x < -1 || x == 1 || x == -1)
            throw new ArgumentException("Check |x| < 1");
        return x / Math.Pow(1 - x, 2);
    }
}
